test/smoke/collapse1/Makefile

```diff
@@ -4,8 +4,8 @@ TESTNAME     = collapse1
 TESTSRC_MAIN = collapse1.c
 TESTSRC_AUX  =
 TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
-RUNENV      += LIBOMPTARGET_KERNEL_TRACE=1
 
+RUNENV      += LIBOMPTARGET_KERNEL_TRACE=1
```

Comment:
Out of curiosity: Is there a possibility that `RUNENV` would be a non-empty value?

Reply:
Good question. I don't know in general. But, if you were to set it inside the same `Makefile` and then add things to it, sure.